src/components/ui/Button/README.md

```diff
@@ -128,19 +158,109 @@ Use a danger button any time you have a destructive action, like deleting someth
 **Changes:** styling
 
 ```jsx
-<Button primary>Active</Button>
-<Button>Hover</Button>
-<Button minimal>Focus</Button>
+<div style={{display: 'flex', flexWrap: 'wrap'}}>
+  <Button primary>Active</Button>
+  <Button>Hover</Button>
+  <Button minimal>Focus</Button>
+</div>
 ```
 
 ```jsx
-<Button primary disabled>
-  Disabled primary
-</Button>
-<Button disabled>Disabled</Button>
-<Button minimal disabled>
-  Minimal disabled
-</Button>
+<div style={{display: 'flex', flexWrap: 'wrap'}}>
+  <Button primary disabled>
+    Disabled primary
+  </Button>
+  <Button disabled>Disabled</Button>
+  <Button minimal disabled>
+    Minimal disabled
+  </Button>
+</div>
 ```
 
 What's my button up to? State will tell you!
+
+### Icon
+
+Use icons in buttons to reinforce meaning. 
```

Comment:
I think this section is a bit out-of-date and not very clear. Could you go back to it and improve this documentation? I think it's a bit disconnected from the code examples.

<img width="522" alt="Screenshot 2020-05-08 16 39 38" src="https://user-images.githubusercontent.com/90871/81422202-90225d00-914a-11ea-9d07-b0a96d6ba82f.png">

Reply:
Agreed! I'll run through the docs and see about improving them prior to merging—they could use a lot of ❤️

Reply:
That was actually definitely some notes to self there. Docs have been reworked, fleshed out, and rearranged now—let me know how it works!